TweetBook/Data/DataContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+using TweetBook.Domain;
+
+namespace TweetBook.Data
+{
+    public class DataContext :DbContext
+    {
+        public DataContext(DbContextOptions<DataContext> options) : base(options)
+        {
+        }
+        public DbSet<Post> Posts { get; set; }
```

Comment:
Any new table you add in the db, you put here in the same way another DbSet. A good thing to remember.